cmd/rook/object/user.go

```diff
@@ -0,0 +1,331 @@
+	"github.com/rook/rook/pkg/model"
+	"github.com/rook/rook/pkg/rook/client"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var (
+	displayNameFlag string
+	emailFlag       string
+)
+
+var userCmd = &cobra.Command{
+	Use:   "user",
+	Short: "Performs commands and operations on object store users in the cluster",
+}
+
+func init() {
+	userCmd.AddCommand(userListCmd)
+	userListCmd.RunE = listUsersEntry
+	userCmd.AddCommand(userGetCmd)
+	userGetCmd.RunE = getUserEntry
+	userCmd.AddCommand(userDeleteCmd)
+	userDeleteCmd.RunE = deleteUserEntry
+	userCmd.AddCommand(userCreateCmd)
+	userCreateCmd.RunE = createUserEntry
+	userCmd.AddCommand(userUpdateCmd)
+	userUpdateCmd.RunE = updateUserEntry
+
+	userCreateCmd.Flags().StringVar(&emailFlag, "email", "", "An email address for the user")
+
+	userUpdateCmd.Flags().StringVar(&emailFlag, "email", "", "An email address for the user")
+	userUpdateCmd.Flags().StringVar(&displayNameFlag, "display-name", "", "A display name for the user")
+}
+
+var userListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Gets a listing with details of all object users in the cluster",
+}
+
+func listUsersEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := listUsers(c)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func listUsers(c client.RookRestClient) (string, error) {
+	users, err := c.ListObjectUsers()
+	if err != nil {
+		return "", fmt.Errorf("failed to get users: %+v", err)
+	}
+
+	if len(users) == 0 {
+		return "", nil
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintln(w, "USER ID\tDISPLAY NAME\tEMAIL")
+
+	for _, b := range users {
+		fmt.Fprintf(w, "%s\t%s\t%s\n", b.UserID, *b.DisplayName, *b.Email)
+	}
+
+	w.Flush()
+	return buffer.String(), nil
+}
+
+var userGetCmd = &cobra.Command{
+	Use:   "get [User ID]",
+	Short: "Gets object user info",
+}
+
+func getUserEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if len(args) == 0 {
+		return fmt.Errorf("Missing required argument User ID")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
+		return err
+	}
+
+	c := rook.NewRookNetworkRestClient()
+	out, err := getUser(c, args[0])
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	fmt.Print(out)
+	return nil
+}
+
+func getUser(c client.RookRestClient, id string) (string, error) {
+	user, err := c.GetObjectUser(id)
+
+	if client.IsHttpNotFound(err) {
+		return "", fmt.Errorf("Unable to find user %s", id)
+	}
+
+	if err != nil {
+		return "", fmt.Errorf("failed to get user: %+v", err)
+	}
+
+	var buffer bytes.Buffer
+	w := rook.NewTableWriter(&buffer)
+
+	fmt.Fprintf(w, "User ID:\t%s\n", user.UserID)
```

Comment:
why not print as a single line with a header line like other commands? Or do we have examples of printing multiple lines for a single object?